polaris.shopify.com/pages/api/v0/tokens/[tokens].tsx

```diff
@@ -67,28 +34,22 @@ function isFormat(format: unknown): format is Format {
   return formats.includes(format as Format);
 }
 
-/**
- * Create the valid github url for a token file.
- */
-export const getGithubUrl = (file: string, isRaw: boolean) => {
-  const fileName = `${file}.json`;
-  const githubUrl = 'https://github.com/Shopify/polaris/blob';
-  const rawUrl = 'https://raw.githubusercontent.com/Shopify/polaris';
-  const filePath = `/main/polaris-react/src/tokens/token-groups/${fileName}`;
-
-  if (isRaw) return `${rawUrl}${filePath}`;
-
-  return `${githubUrl}${filePath}`;
-};
-
 /**
  * Format the token data into: css or json
  */
 const formatTokenGroup = (tokenGroup: TokenGroup, format: Format) => {
+  const tokenValues = Object.fromEntries(
+    Object.entries(tokenGroup).map((entry) => {
+      const [key, tokenProps] = entry as [TokenGroupKey, TokenProperties];
+
+      return [key, tokenProps.value];
+    }),
+  );
+
   if (format === 'css') {
-    return Object.keys(tokenGroup)
+    return Object.keys(tokenValues)
       .reduce<string[]>((result, token) => {
-        const cssVariable = `--${token}: ${tokenGroup[token]};`;
+        const cssVariable = `--${token}: ${tokenValues[token]};`;
```

Comment:
```suggestion
        const cssVariable = `--p-${token}: ${tokenValues[token]};`;
```

Should this have a `p-`? I think it's time we create a `getVar` or `createVar` util that accepts a `tokenName` and generates the `--p-<tokenName>` string. Happy for that to be in a separate PR, but we are doing that transformation in multiple places and would be nice to have an established pattern.

Reply:
Created https://github.com/Shopify/polaris/pull/5540